addons/mrp/models/mrp_workcenter.py

```diff
@@ -452,6 +453,26 @@ def _loss_type_change(self):
         else:
             self.loss_id = self.env.ref("mrp.block_reason7").id
 
+    def create(self, values):
+        if not isinstance(values, list) or not values[-1].get("workorder_id"):
+            return super(MrpWorkcenterProductivity, self).create(values)
+        wo = self.env['mrp.workorder'].browse(values[-1]["workorder_id"])
```

Comment:
Are we absolutely sure that if the values are a list, it will be the last one that will be the one to split?
Could the list of values be filled with more than one new timesheets?